rotkehlchen/accounting/structures.py

```diff
@@ -312,6 +312,46 @@ class HistoryEventSubType(SerializableEnumMixin):
     SPEND = 6
     RECEIVE = 7
 
+    def serialize_event_subtype(self) -> str:
```

Comment:
Why are you changing the name and not simply overwriting serialize()?

Reply:
because the method makes a standard serialization across the code and didn't see like a good practice to break this in the code. Also the serialized function is used to store the events in database